### DISCUSSION TITLE
i have a problem installing tsserver

### DISCUSSION THREAD

**Discussion by DualSeven** [neoclide/coc.nvim#3966] [Q&A] [answered]
i wanna know
coc-Tsserver is'n installing appears a X red.

 i look on web but nothing maybe is someting simple.. help me.. 

![Screenshot_20220801-130609_Termux.jpg](https://user-images.githubusercontent.com/98855047/182197832-752b54d6-c525-4873-8f78-74eac48b4188.jpg)

**Comment by chemzqm** [comment, marked as answer]
Should be network issue or maybe your phone out of memory.

**Reply by DualSeven** [reply]
maybe a network because i have an samsung-A70_6gb Ram.. maybe can be the link? well.. thanks
